rhel/SPECS/pmm-server.spec

```diff
@@ -56,7 +56,6 @@ mv .htpasswd  %{buildroot}%{_sysconfdir}/nginx/.htpasswd
 mv nginx.conf %{buildroot}%{_sysconfdir}/nginx/conf.d/pmm.conf
 mv nginx-ssl.conf %{buildroot}%{_sysconfdir}/nginx/conf.d/pmm-ssl.conf
 install -d %{buildroot}%{_sysconfdir}/cron.daily
-mv purge-qan-data %{buildroot}%{_sysconfdir}/cron.daily/purge-qan-data
```

Comment:
Please bump the release https://github.com/percona/pmm-server-packaging/blob/be8dd71429546a25a56feb7d7c493b4b5c652c02/rhel/SPECS/pmm-server.spec#L14